owa/app/js/components/manage_patient_identifier_sources/edit_patient_identifier_sources/localPoolIdentifiers.jsx

```diff
@@ -126,10 +126,11 @@ export default class EditLocalPoolIdentifiers extends Component {
             <tr>
               <td className="editPatientIdentifierSourcesTitle">Pool Identifier Source:</td>
               <td>
-                <Input id="sourceUuid" name="sourceUuid" value={this.state.sourceUuid} onChange={this.handleSelectChange} className="editPatientIdentifierSourcesInput" size="sm" type="select">
+                <Input id="sourceUuid" name="sourceUuid" value={this.state.sourceUuid ? this.state.sourceUuid : ""} onChange={this.handleSelectChange} className="editPatientIdentifierSourcesInput" size="sm" type="select">
```

Comment:
This line is too long. Consider breaking it into smaller chunks on multiple lines.

Reply:
fixed